src/network/udp.py

```diff
@@ -1,27 +1,33 @@
-import time
-import Queue
+"""
+src/network/udp.py
+==================
+
+"""
+
 import socket
+import time
 
+import protocol
+import state
 from debug import logger
 from network.advanceddispatcher import AdvancedDispatcher
-from network.bmproto import BMProtoError, BMProtoInsufficientDataError, BMProto
-from network.bmobject import BMObject, BMObjectInsufficientPOWError, BMObjectInvalidDataError, BMObjectExpiredError, BMObjectInvalidError, BMObjectAlreadyHaveError
-import network.asyncore_pollchoose as asyncore
+from network.bmproto import BMProto
 from network.objectracker import ObjectTracker
+from queues import receiveDataQueue
 
-from queues import objectProcessorQueue, UISignalQueue, receiveDataQueue
-import state
-import protocol
 
 class UDPSocket(BMProto):
+    """"""
+    # pylint: disable=too-many-instance-attributes
+
     port = 8444
     announceInterval = 60
 
     def __init__(self, host=None, sock=None, announcing=False):
-        super(BMProto, self).__init__(sock=sock)
+        super(UDPSocket, self).__init__(sock=sock)
```

Comment:
Did you test? This may be the `__init__` of `AdvancedDispatcher` or `ObjectTracker`.

Reply:
Good catch, this needs to be tested due to multiple inheritence.